rpms/s3/s3rpm.spec

```diff
@@ -441,8 +440,16 @@ sh /opt/seagate/cortx/auth/scripts/swupdate/merge.sh
 systemctl daemon-reload
 systemctl enable s3authserver
 systemctl restart rsyslog
+
+# Create soft link for 's3_setup' in '/usr/local/bin'
+ln -sf /opt/seagate/cortx/s3/bin/s3_setup /usr/local/bin/s3_setup
+chmod 755 /usr/local/bin/s3_setup
+
 openssl_version=`rpm -q --queryformat '%{VERSION}' openssl`
 if [ "$openssl_version" != "1.0.2k" ] && [ "$openssl_version" != "1.1.1" ]; then
   echo "Warning: Unsupported (untested) openssl version [$openssl_version] is installed which may work."
   echo "Supported openssl versions are [1.0.2k, 1.1.1]"
 fi
+
+%postun
```

Comment:
spelling mistake .. may be "%postrun"

Reply:
I checked, its %postun.

Reply:
yes. reference link : https://rpm-packaging-guide.github.io/#triggers-and-scriptlets